bofire/strategies/predictives/botorch.py

```diff
@@ -326,71 +431,58 @@ def _ask(self, candidate_count: int) -> pd.DataFrame:
 
         (
             bounds,
```

Comment:
`bounds` are the global bounds in contrast to local bounds, right?

Reply:
Yes!